library/Zend/Stdlib/Hydrator/Strategy/StrategyChain.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
@samsonasik I'd rather change these on merge instead of poking each author of each PR.